lib/internal/streams/from.js

```diff
@@ -4,13 +4,25 @@ const {
   Symbol,
   SymbolIterator
 } = primordials;
+const { Buffer } = require('buffer');
 
 const {
   ERR_INVALID_ARG_TYPE
 } = require('internal/errors').codes;
 
 function from(Readable, iterable, opts) {
   let iterator;
+  if (typeof iterable === 'string' || iterable instanceof Buffer) {
```

Comment:
Should we check for `Stream._isUint8Array` as well?

Reply:
That would only be okay if we turn off object mode, I think